weblate/templates/snippets/project/billing-libre-trial.html

```diff
@@ -6,8 +6,8 @@
 {% block message %}
   {% comment %}user_can_view_billing is defined in snippets/project/state.html{% endcomment %}
   {% if user_can_view_billing %}
-    <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for libre hosting" %}</a>
+    <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for Libre hosting" %}</a>
   {% endif %}
-  {% trans "This project is in the trial period, so be cautious before you contribute. Things like license or repository can still change before being approved as a libre project." %}
+  {% trans "Hey translator! This project is in a trial period, and its setup might not be complete yet!</br>Consider asking its maintainers before you contribute to avoid the chance of losing your work. Remember that before approval as a libre project, essential things like a license can change." %}
```

Comment:
Let's remove the HTML from the string to make it easier to handle when translating:

```suggestion
  {% trans "Hey translator! This project is in a trial period, and its setup might not be complete yet!" %}
  <br/>
  {%trans "Consider asking its maintainers before you contribute to avoid the chance of losing your work. Remember that before approval as a libre project, essential things like a license can change." %}
```

Reply:
Thanks!